apps/rush/src/RushCommandSelector.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as path from 'path';
+import * as myRushLib from '@microsoft/rush-lib';
+
+type CommandName = 'rush' | 'rushx' | undefined;
+
```

Comment:
enum?

Reply:
RIght now this is just a private definition.  We can make it an enum later if it gets more formalized.

---
In reply to: [193537077](https://github.com/Microsoft/web-build-tools/pull/678#discussion_r193537077) [](ancestors = 193537077)